frontend/src/metabase/admin/datamodel/selectors.js

```diff
@@ -3,17 +3,17 @@ import { createSelector } from 'reselect';
 import { computeMetadataStrength } from "metabase/lib/schema_metadata";
 
 
-const segmentsSelector         = state => state.datamodel.segments;
-const metricsSelector          = state => state.datamodel.metrics;
+const segmentsSelector         = (state, props) => state.datamodel.segments;
```

Comment:
Possibly overkill, but pretty harmless, and it was getting annoying trying to track down which ones needed `props`, especially in `mapStateToProps`. I suppose it's not at all necessary for the "leaf" selectors though.